lib/ui/dart_wrapper.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_LIB_UI_DART_WRAPPER_H_
+#define FLUTTER_LIB_UI_DART_WRAPPER_H_
+
+#include "lib/fxl/memory/ref_counted.h"
+#include "third_party/tonic/dart_wrappable.h"
+
+template <typename T>
+class RefCountedDartWrappable : public fxl::RefCountedThreadSafe<T>,
```

Comment:
Can we move to the fml variant while we are at it anyway?

Reply:
I'd like to defer this to a future PR

Reply:
sgtm.